cores/esp8266/Esp.h

```diff
@@ -123,12 +123,12 @@ class EspClass {
 
 #if defined(F_CPU) || defined(CORE_MOCK)
         constexpr uint8_t getCpuFreqMHz() const
+#else
+        uint8_t getCpuFreqMHz() const
+#endif
```

Comment:
Minor nitpick: please put the constexpr specifier on its own line here. It took me several seconds to understand that this is alternates signatures.
E.g.:
```
#if defined(F_CPU) || defined(CORE_MOCK)
        constexpr 
        uint8_t getCpuFreqMHz() const
#else
        uint8_t getCpuFreqMHz() const
#endif
```
or maybe this:
```cpp
#if defined(F_CPU) || defined(CORE_MOCK)
        constexpr 
#endif
        uint8_t getCpuFreqMHz() const
```

Reply:
You are right.